pkg/internal/utils/utils.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package utils
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	policyapi "sigs.k8s.io/network-policy-api/pkg/client/clientset/versioned"
+
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/eval"
+)
+
+// this internal file contains code which is consumed by the netpol-analyzer commands in cli and netpol packages
+const CtxTimeoutSeconds = 3
+
+// UpdatePolicyEngineWithK8sPolicyAPIObjects inserts to the policy-engine all (baseline)admin network policies
+func UpdatePolicyEngineWithK8sPolicyAPIObjects(pe *eval.PolicyEngine, clientset *policyapi.Clientset) error {
```

Comment:
why is this a function of `utils` and not of a method of `PolicyEngine`?

Reply:
right, changing